### ISSUE TITLE
Contributing Made fun & Easy

### ISSUE THREAD

**Issue by murillol** [wpmarketingteam/Marcomms-Onboarding#7]
If “Contributing (more) to the Global WordPress Marketing Team” happens to be on your list of potential new year's resolutions, we have great news for you. We are super excited to start a series of onboarding sessions for new contributors called “Contributing Made Fun & Easy”. 

Join us and discover how to best contribute your talents, get to know the tools we use, and adopt good practices for working remotely and asynchronously.

At present, this is going to be a four-part series that will take place on a biweekly schedule. 

The first session will take place on **Tuesday, January 19th, 2021, at 11:00 am UTC**. These training sessions are aimed to help both newcomers and those who would like to get a refresher to get on board with the team and the tasks that are underway. The Global WordPress Marketing team continuously makes an effort to market WordPress to the outer world. Also, we’re an inclusive team, working hard to make everyone feel comfortable and welcomed.
The initial session starts off with basics covering details about: 

-The WordPress project
-Its code of conduct
-Getting to know each other
-How marketing team works and its tasks
-Setting up your WordPress profile

**SIGN UP FORM**
`https://docs.google.com/forms/d/e/1FAIpQLSdHA3b_n0GTRRuqPdpeRl8RLE2-MUv_cdv743DssMeYQ0Ly2w/viewform`

If you have any questions, do not hesitate to reach out to the following team members through the WordPress Slack Marketing channel: @maedahbatool and @yvettesonneveld

**Comment by murillol** [comment]
Checklist SESSION 1: Tuesday, January 19th, 2021, at 11:00 am UTC

- [ ] The WordPress project
- [ ] Its code of conduct
- [ ] Getting to know each other
- [ ] How marketing team works and its tasks
- [ ] Setting up your WordPress profile

Copied from Trello: https://trello.com/c/aMoGsXLq

**Comment by TantienHime** [comment]
@murillol Was this a record of a previous item? Notice you are using 2021 here. Should it be 2022?

**Comment by murillol** [comment]
> @murillol Was this a record of a previous item? Notice you are using 2021 here. Should it be 2022?

Yes! It's a record of a previous item that we moved from Trello. The initial workshops already took place but I believe this is meant to be an ongoing activity, so I'm sure Yvette and Maedah will update when they get a chance :-)

**Comment by murillol** [comment]
> @murillol Was this a record of a previous item? Notice you are using 2021 here. Should it be 2022?

Yes! It's a record of a previous item that we moved from Trello. The initial workshops already took place but I believe this is meant to be an ongoing activity, so I'm sure Yvette and Maedah will update when they get a chance :-)
